project-base/docker/php-fpm/Dockerfile

```diff
@@ -31,12 +31,17 @@ RUN chmod +x /usr/local/bin/docker-install-composer && \
 # autoconf needed by "redis" extension
 RUN apt-get install -y \
     libpng-dev \
+    libjpeg-dev \
+    libfreetype6-dev \
     libzip-dev \
     libicu-dev \
     libpq-dev \
     autoconf && \
     apt-get clean
 
+# "gd" extension needs to have specified jpeg and freetype dir for jpg/jpeg images support
+RUN docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/
```

Comment:
and please why do we need to specify directories and why is it `/usr/include`? because this directory contains more than just jpeg support

Reply:
http://php.net/manual/en/image.installation.php based on manual